jtravis/src/main/java/fr/inria/spirals/jtravis/helpers/RepositoryHelper.java

```diff
@@ -30,31 +32,49 @@ protected static RepositoryHelper getInstance() {
     }
 
     public static Repository getRepositoryFromSlug(String slug) {
-        String resourceUrl = getInstance().getEndpoint()+REPO_ENDPOINT+slug;
-
+        String resourceUrl ="";
+        if(Version.getVersionV3()) {
+            String urlSlug = "";
+            try {
+                urlSlug = URLEncoder.encode(slug, "UTF-8");
```

Comment:
Basically you need to do it each time you use V3 and put a slug name, so extract this try/catch to an utility function.